x-pack/test/api_integration/deployment_agnostic/apis/observability/apm/data_view/static.spec.ts

```diff
@@ -42,33 +42,43 @@ export default function ApiTest({ getService }: DeploymentAgnosticFtrProviderCon
   }
 
   describe('Data view static', () => {
-    let supertest: SupertestWithRoleScope;
+    let supertestEditorWithApiKey: SupertestWithRoleScope;
+    let supertestEditorWithCookieCredentials: SupertestWithRoleScope;
 
     before(async () => {
-      supertest = await roleScopedSupertest.getSupertestWithRoleScope('editor', {
+      supertestEditorWithApiKey = await roleScopedSupertest.getSupertestWithRoleScope('editor', {
         withInternalHeaders: true,
       });
+
+      supertestEditorWithCookieCredentials = await roleScopedSupertest.getSupertestWithRoleScope(
+        'editor',
+        {
+          useCookieHeader: true,
+          withInternalHeaders: true,
+        }
+      );
     });
 
     after(async () => {
-      await supertest.destroy();
+      await supertestEditorWithApiKey.destroy();
+      await supertestEditorWithCookieCredentials.destroy();
```

Comment:
Sorry, I should have mentioned it before: Cookie headers persist for the whole test run (cached since it is validity is ~20 min but when you use it is auto-prolonged)

tl; dr; `await supertestEditorWithCookieCredentials.destroy();` can be removed as it won't do anything.

Reply:
done!